Softeq.XToolkit.PushNotifications/Utils/TaskExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Softeq.XToolkit.PushNotifications.Utils
+{
+    // TEMPORARY COPY
+    internal static class TaskExtensions
```

Comment:
Please, you can use XToolkit.Common infrastructure

Reply:
Done